triton-client/src/components/ReadsPerSample/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { ReactElement, useCallback, useEffect } from "react"
+import ReadsPerSampleProps, { ReadsPerSampleButtonProps } from "./interfaces"
+import { useAppSelector } from "@store/hooks"
+import { selectReadsPerSample } from "@store/selectors"
+import { fetchReadsPerSample } from "@store/thunks"
+import { store } from "@store/store"
+import { Button, Modal, Spin, Typography } from "antd"
+import { InfoCircleOutlined } from "@ant-design/icons"
+import { Provider } from "react-redux"
+import ReadsPerSampleGraph from "@components/ReadsPerSampleGraph"
```

Comment:
overall, the code looks clean but you didn't separate the components in this folder which makes the imports of this component different from the others components in other files.  
1. i would say to have each of the components in their own file/folder -> export default -> update @components/index.ts
2. optional, make a generic button component in order to simply feed the data/info to it